### ISSUE TITLE
Converter for scikit-learn IsolationForest ??? 

### ISSUE THREAD

**Issue by mehdilkotbi** [onnx/sklearn-onnx#234]
Hello, 
Have anyone tried to implement a converter for scikit-learn's Isolation Forest? I tried to use Random Forest's converter to create one since it's the closest to Isolation Forest but it didn't work. If any one has any idea, please share! 
Thanks.
Mehdi.

**Comment by xadupre** [comment]
The prediction function requires to extract the decision path and this requires a change in ONNX spécifications.

**Comment by xadupre** [comment]
Maybe it can be done by changing the predicted label. To be investigated.
